infrastructure/tests/test_data_import.py

```diff
@@ -124,3 +196,114 @@ def test_year_error(self):
+        self.assertEquals(project.own_strategic_objectives, "TO BE CORRECTED")
+        self.assertEquals(project.asset_class, "Furniture and Office")
+        self.assertEquals(project.asset_subclass, "Equipment")
+        self.assertEquals(project.ward_location, "Head Office")
+        self.assertEquals(project.longitude, 1.0)
+        self.assertEquals(project.latitude, 2.0)
+
+        self.assertEquals(Expenditure.objects.all().count(), 8)
+        verify_expenditure(self, 2100.00, "Audited Outcome", "2018/2019")
+        verify_expenditure(self, 3100.00, "Full Year Forecast", "2019/2020")
+        verify_expenditure(self, 6100.00, "Budget year", "2022/2023")
+
+
+    def test_new_project(self):
+        """Scope of Test: With an existing project import a new project with a new composite key and check that a new  project is created"""
+
+        geography = Geography.objects.get(geo_code="BUF")
+        utils.load_file(geography, mock_project_row(), "2019/2020")
+
+        project = Project.objects.get(project_description="P-CNIN FURN & OFF EQUIP")
+        self.assertEquals(project.project_description, "P-CNIN FURN & OFF EQUIP")
+        self.assertEquals(project.project_number, "PC002003005_00002")
+        self.assertEquals(project.project_type, "New")
+        self.assertEquals(project.mtsf_service_outcome, "An efficient, effective and development-oriented public service")
+        self.assertEquals(project.iudf, "Growth")
+        self.assertEquals(project.own_strategic_objectives, "OWN MUNICIPAL STRATEGIC OBJECTIVE")
+        self.assertEquals(project.asset_class, "Furniture and Office Equipment")
+        self.assertEquals(project.asset_subclass, "")
+        self.assertEquals(project.ward_location, "Administrative or Head Office")
+        self.assertEquals(project.longitude, 0.0)
+        self.assertEquals(project.latitude, 0.0)
+
+        utils.load_file(geography, mock_new_project_row(), "2020/2021")
+        self.assertEquals(Project.objects.all().count(), 2)
+
+        project = Project.objects.get(project_description="P-CNIN FURN & OFF EQUIP - NEW DESCRIPTION")
+        self.assertEquals(project.project_type, "Renewal")
+        self.assertEquals(project.mtsf_service_outcome, "A project update")
+        self.assertEquals(project.iudf, "Governance")
+        self.assertEquals(project.own_strategic_objectives, "TO BE CORRECTED")
+        self.assertEquals(project.asset_class, "Furniture and Office")
+        self.assertEquals(project.asset_subclass, "Equipment")
+        self.assertEquals(project.ward_location, "Head Office")
+        self.assertEquals(project.longitude, 1.0)
+        self.assertEquals(project.latitude, 2.0)
+
+        self.assertEquals(Expenditure.objects.all().count(), 10)
+
+        expenditure_id = Expenditure.objects.get(amount=2100).project_id
+        new_expenditures = Expenditure.objects.filter(project_id=expenditure_id)
+        self.assertEquals(new_expenditures.count(), 5)
```

Comment:
This would have been more readable if you named the original project initial_project and the second project new_project and then did 
```
         new_expenditures = Expenditure.objects.filter(project=new_project)
```